mods/ra/rules/infantry.yaml

```diff
@@ -158,6 +162,8 @@ E4:
 	Armament@GARRISONED:
 		Name: garrisoned
 		Weapon: Flamer
+	TakeCover:
+		ProneOffset: 500,0,-341
```

Comment:
This is quite wrong, and I just noticed that the standing fire offsets are wrong too:

![screen shot 2017-02-04 at 11 19 11](https://cloud.githubusercontent.com/assets/167819/22618079/ef70374c-eacb-11e6-8af7-557c7b25e9c6.png)
"It burns when I pee!"